.gitignore

```diff
@@ -147,9 +147,3 @@ yarn.lock
 # editor files
 .vscode
 .idea
-
-#tap files
-.tap/
-
-# test tap report
-out.tap
```

Comment:
Best to keep these in, just incase someone has old tap tests.

Reply:
I have actually removed all of them in the other migrations I did, wouldn't be better to keep things aligned?

Reply:
Roger, thank you so much @Fdawgs for your patience

Reply:
Ah balls, didn't spot that.